Sources/StreamChat/APIClient/Endpoints/Requests/CastPollVoteRequestBody.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct CastPollVoteRequestBody: Encodable {
+    let pollId: String
+    var vote: VoteDataRequestBody?
+
+    init(pollId: String, vote: VoteDataRequestBody? = nil) {
+        self.pollId = pollId
+        self.vote = vote
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case pollId = "poll_id"
+        case vote
+    }
+}
+
+struct VoteDataRequestBody: Encodable {
+    var answerText: String?
+    var optionId: String?
+    var option: PollVoteOptionRequestBody?
+
+    init(
+        answerText: String? = nil,
+        optionId: String? = nil,
+        option: PollVoteOptionRequestBody? = nil
+    ) {
+        self.answerText = answerText
+        self.optionId = optionId
+        self.option = option
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case answerText = "answer_text"
+        case optionId = "option_id"
+        case option = "Option"
```

Comment:
I guess someone make a mistake on the backend side with capitalisation

Reply:
It's correct with capital O: https://getstream.io/chat/docs/rest/#polls-castpollvote. 😁